README.md

```diff
@@ -97,3 +97,18 @@ fn jump(query: Query<&ActionState<Action>, With<Player>>) {
 ```
 
 This snippet is the `minimal.rs` example from the [`examples`](./examples) folder: check there for more in-depth learning materials!
+
+## Crate Feature Flags
+
+The following crate feature flags are available. They are configured in your `Cargo.toml`.
```

Comment:
We should avoid duplicating this information. Can you instead link to the `Cargo.toml` here?